composer.json

```diff
@@ -34,7 +34,9 @@
 		"tracy/tracy": "Diagnostics tool from Nette",
 		"symfony/locale": "Locale component provides fallback code to handle cases when the intl extension is missing.",
 		"kdyby/console": "If you wanna use extract command and much others, install also console.",
-		"symfony/yaml": "If you wanna store translations in YAML format - supports multiline strings."
+		"symfony/yaml": "If you wanna store translations in YAML format - supports multiline strings.",
+	  	"kdyby/doctrine": "If you wanna store translations in database and use Doctrine.",
```

Comment:
v další fázi plánuji přidat orm mapování téhle překladové tabulky, aby si doctrine migrations myslely, že jde o entitu a nesnažily se ji smazat. Na to potřebuji kdyby doctrine.